earthaccess/api.py

```diff
@@ -283,8 +293,10 @@ def download(
 def open(
     granules: Union[List[str], List[DataGranule]],
     provider: Optional[str] = None,
+    credentials_endpoint: Optional[str] = None,
     *,
     pqdm_kwargs: Optional[Mapping[str, Any]] = None,
+    open_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
I suggest `ffspec_open_kwargs` instead. If we switch to or add support for obstore in the future, it would be easier to document and migrate if the parameters had different names.

Reply:
Since Obstore is not a POSIX file system wrapper(caveats), there is no `.open(URL)` , I tihnk `client_options`  would be the closest to `open_kwargs` in fsspec, I can rename it for now and down the road we could just go one level up and do something like "io_driver" and then assume that we can get range requests and all of the operations an xarray backend expects and both will share a `driver_kwargs` cc @owenlittlejohns @mfisher87